.github/workflows/build.yml

```diff
@@ -205,9 +232,36 @@ jobs:
         GH_REPO: ${{ github.repository }}
       run: gh release upload "${{ needs.release.outputs.tag }}" "ruby-${{ matrix.name }}-${{ steps.platform.outputs.platform }}.tar.gz"
 
+  reuse-slow:
+    needs: [prepare, release]
+    strategy:
+      fail-fast: false
+      matrix: ${{ fromJson(needs.prepare.outputs.reuse_matrix) }}
+    runs-on: ${{ matrix.os }}
+    env:
+      GH_TOKEN: ${{ github.token }}
+      GH_REPO: ${{ github.repository }}
+    steps:
+      - name: Set platform
+        id: platform
+        run: |
+          platform=${{ matrix.os }}
+          platform=${platform/macos-13/macos-latest}
+          platform=${platform/macos-14/macos-13-arm64}
+          platform=${platform/%-arm/-arm64}
+          echo "platform=$platform" >> $GITHUB_OUTPUT
+      - name: Download binaries from previous release
+        run: gh release download "${{ needs.prepare.outputs.previous_release }}" --pattern "ruby-${{ matrix.name }}-${{ steps.platform.outputs.platform }}.tar.gz"
```

Comment:
No need to download `if: ${{ github.event_name == 'workflow_dispatch' && github.event.inputs.skip_slow == 'true' }}` here

Reply:
Yeap -- I noticed that myself, fixed in 809d82a2150712ac9ed982cd748a5676275f2c83